index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+ <nav class="container">
+  <div class="img">
+  <img src="images/logo.svg" alt="">
+  </div>
+  <div class="navbar">
+    <ul>
+      <li class="item"><a href="#">Home</a></li>
+      <li class="item"><a href="#">New</a></li>
+      <li class="item"><a href="#">Popular</a></li>
+      <li class="item"><a href="#">Trending</a</li>
+      <li class="item"><a href="#">Cartegories</a></li>
+    </ul>
```

Comment:
kindly ensure that you link the following lists for the responsiveness
```suggestion
    <ul>
      <li class="item"><a href="#">Home</a></li>
      <li class="item"><a href="#">New</a></li>
      <li class="item"><a href="#">Popular</a></li>
      <li class="item"><a href="#">Trending</a</li>
      <li class="item"><a href="#">Cartegories</a></li>
    </ul>
    

```